code/WorkInProgress/parallax.dm

```diff
@@ -0,0 +1,214 @@
+#define PARALLAX_VIEW_WIDTH (WIDE_TILE_WIDTH + 4) // stolen from pali
+#define PARALLAX_VIEW_HEIGHT (SQUARE_TILE_WIDTH + 4)
+
+/datum/parallax_object // stores all the objects for parallax that arent the background
```

Comment:
you keep using normal comments instead of code doc comments
use code doc comments